account_page.php

```diff
@@ -86,12 +86,19 @@
 
 html_page_top( lang_get( 'account_link' ) );
 
+$t_user_id = auth_get_current_user_id();
+
 # extracts the user information for the currently logged in user
 # and prefixes it with u_
-$t_row = user_get_row( auth_get_current_user_id() );
+$t_row = user_get_row( $t_user_id );
 
 extract( $t_row, EXTR_PREFIX_ALL, 'u' );
 
+if ( is_blank( $u_token ) ) {
+	$u_token = helper_generate_uuid();
+	user_set_field( $t_user_id, 'token', $u_token );
```

Comment:
The cookie may have a limited lifetime in the browser (1 session or 1 year in case of "remember me"), but the value of the cookie is valid and doesn't change.  So if user gets it from the database, then they can login as the user without issue.  We only rotate the value of the cookie when the user changes their password, hence, causing them to be logged out of all browsers.  Hence, there is no difference in terms of access I can get between the token vs. cookie except that cookie gives me interactive web access and token gives me API access.